### PR TITLE
feat(backend): Implement route to update is_public status for all ContributorRating

### PR DESCRIPTION
**Pull request by amatissart**
Related to #274 

The new route `PATCH /users/me/contributor_ratings/_all/`  will be used on the page "My rated videos" to allow marking of all existing user contributions as "public" or "private".

### CODE REVIEW

**Review by GresilleSiffle — APPROVED** [review]
LGTM

File: backend/tournesol/urls.py [hunk 1]
```diff
@@ -70,6 +74,11 @@
         ContributorRatingList.as_view(),
         name="ratings_me_list",
     ),
+    path(
+        "users/me/contributor_ratings/_all/",
```
Comment (lfaucon) [inline backend/tournesol/urls.py:78]: Do we need this route? Can't we just use the patch on the route `"users/me/contributor_ratings/"` and rely on the ContributorRatingList view?

Comment (amatissart) [reply backend/tournesol/urls.py:78]: I actually hesitated with this solution. Implementing `PATCH` on a collection of resources does not seem to be a very common pattern in RESTful APIs. And DRF mixins are not suitable for this case, so a custom implementation would be necessary anyway.

I also find that defining a special identifier makes the intent a bit more explicit: `_all` can be PATCHed just like any other instance. And any future action/route on individual ratings could also be defined for this identifier too.

But it's mostly a matter of taste. If you prefer to stick closer to "standard" REST I can change it.

Comment (lfaucon) [reply backend/tournesol/urls.py:78]: Sounds good! Let's go with your initial proposition 👍

**Review by lfaucon — APPROVED** [review]

**Pull request merged**